code_to_gast/loop/py_loop.py

```diff
@@ -1,8 +1,91 @@
 import py_router as pr
+import ast
 
 def while_statement_to_gast(node):
     gast = {}
     gast["type"] = "whileStatement"
     gast["body"] = pr.node_to_gast(node.body)
     gast["test"] = pr.node_to_gast(node.test)
+    return gast
+
+def for_statement_to_gast(node):
+    # first identify whether forRange or forOf
+    if type(node.iter) == ast.Call:
+        return for_range_statement_to_gast(node)
+    else: 
+        return for_of_statement_to_gast(node)
+
+def for_of_statement_to_gast(node):
+    gast = {}
+    gast["type"] = "forOfStatement"
+    gast["init"] = pr.node_to_gast(node.target)
+    gast["iter"] = pr.node_to_gast(node.iter)
+    gast["body"] = pr.node_to_gast(node.body)
+    return gast
+
+
+def for_range_statement_to_gast(node):
+    gast = {}
+    gast["type"] = "forRangeStatement"
+
+    args = node.iter.args
+    if len(args) < 2 or len(args) > 3:
```

Comment:
Maybe add a comment here about why <2 and >3 is unsupported

Reply:
+1 - would save us a little brain power :) 

Or even better move this check to a function with a descriptive name. Generally... I trust function names more than comments if you can sufficiently capture the reason in a function name.